src/Catalyst.Protocol.Tests/UnitTests/Wire/FavouriteDeltaBroadcastTests.cs

```diff
@@ -0,0 +1,106 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using Catalyst.Core.Lib.Extensions;
+using Catalyst.Protocol.Peer;
+using Catalyst.Protocol.Wire;
+using FluentAssertions;
+using Google.Protobuf;
+using Xunit;
+using CandidateDeltaBroadcast = Catalyst.Protocol.Wire.CandidateDeltaBroadcast;
+
+namespace Catalyst.Protocol.Tests.Wire
+{
+    public class FavouriteDeltaBroadcastTests
+    {
+        private sealed class InvalidFavouriteDeltaBroadcasts : TheoryData<FavouriteDeltaBroadcast>
+        {
+            public InvalidFavouriteDeltaBroadcasts()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private constructor 'InvalidFavouriteDeltaBroadcasts'.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4318201)